src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AddNewUserTest.java

```diff
@@ -16,15 +19,40 @@
 @ActiveProfiles("functional")
 public class AddNewUserTest extends AuthorizationFunctionalTest {
 
+    String orgIdentifierResponse = null;
+
+    @Before
+    public void createAndUpdateOrganisation() {
+        orgIdentifierResponse = createAndUpdateOrganisationToActive(hmctsAdmin);
+    }
 
     @Test
     public void add_new_user_to_organisation() {
 
-        String organisationIdentifier = createAndUpdateOrganisationToActive(hmctsAdmin);
+        NewUserCreationRequest newUserCreationRequest = professionalApiClient.createNewUserRequest();
+        Map<String, Object> newUserResponse = professionalApiClient.addNewUserToAnOrganisation(orgIdentifierResponse, hmctsAdmin,  newUserCreationRequest, HttpStatus.CREATED);
+        assertThat(newUserResponse).isNotNull();
+    }
+
+    @Test
+    public void add_new_user_to_organisation_with_no_jurisdiction_should_return_400() {
 

```

Comment:
is this 400 being thrown from prd or sidam?

as per requirement  from Jira -
PRD will therefore not validate if any users, with 'Zero/None' jurisdiction IDs (i.e. without any Jurisdiction ID), are requested to be added to organisations as the onus for validation will rest with ExUI and SIDAM

Reply:
CCD returns back 400 to PRD and hence we are propagating same. We ave removed validation for jurisdiction which used to check validate jurisdictions and at least 1 element.

Reply:
👍